scripts/Muon/GUI/Common/plot_widget/plot_widget_presenter.py

```diff
@@ -339,14 +340,13 @@ def match_raw_selection(self, workspace_names, plot_raw):
                 ws_list.append(workspace_name)
         return ws_list
 
-    def handle_plot_guess_changed(self):
-        if self.context.fitting_context.guess_ws is None:
-            return
+    def handle_remove_plot_guess(self):
+        if self.context.fitting_context.guess_workspace_name is not None:
+            self._figure_presenter.remove_workspace_names_from_plot([self.context.fitting_context.guess_workspace_name])
 
-        if self.context.fitting_context.plot_guess:
-            self._figure_presenter.plot_guess_workspace(self.context.fitting_context.guess_ws)
-        else:
-            self._figure_presenter.remove_workspace_names_from_plot([self.context.fitting_context.guess_ws])
+    def handle_update_plot_guess(self):
+        if self.context.fitting_context.guess_workspace_name is not None and self.context.fitting_context.plot_guess:
+            self._figure_presenter.plot_guess_workspace(self.context.fitting_context.guess_workspace_name)
```

Comment:
This looks ok to me, and is not a github rendering issue

Reply:
I think it is just how github is showing me the code when I review it via Files Changed! It looks fine here but this is what I see in Files Changed where ```self._figure_presenter.plot_guess_workspace(self.context.fitting_context.guess_workspace_name)``` looks like it has no indentation at all.

![Screenshot (142)](https://user-images.githubusercontent.com/55837273/121510460-8c14de80-c9df-11eb-981c-b6c7e7116a3c.png)

Reply:
Ah ok, yes this is happening because of the way github is rendering it. When it does this it helps to look at the line numbers on the left hand side, as you can see its all on line 348